src/hooks/RefreshTime.ts

```diff
@@ -0,0 +1,26 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { consts } from '../config';
+import { refreshTimeFor } from '../helpers';
+
+export const useRefreshTime = (
+  refreshTimeKey: string,
+  refreshInterval = consts.REFRESH_INTERVALS.STATIC_CONTENT
```

Comment:
maybe we should export `const ONCE_A_DAY = 'ONCE_A_DAY';` in `consts` to use it here, as `STATIC_CONTENT` is logically used for data of type static contents. the value is the same, but the meaning is a different.

Reply:
done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/142/commits/b525e8503899998113202a45cec465cccab2b50a